docs/dictionary/function/revXMLRPC_GetHost.lcdoc

```diff
@@ -18,39 +18,40 @@ Platforms: desktop, server, mobile
 Security: disk, process
 
 Example:
-put revXMLRPC_GetHost(theRequest) into theHost
+local tHost
+put revXMLRPC_GetHost(tRequest) into tHost
 
 Parameters:
-documentID:
-The number returned by the revXMLRPC_CreateRequest when you created the
-XML-RPC request, or the successful revXMLRPC_Execute function when you
-executed an XML-RPC request.
+XMLRPCdocumentID (integer):
+The number returned by the <revXMLRPC_CreateRequest> when you created the
+<XML-RPC> request, or the successful <revXMLRPC_Execute> <function> when you
+executed an <XML-RPC> request.
 
-Returns:
+Returns (string):
 The <revXMLRPC_GetHost> <function> returns the <IP address> or <domain
 name> of the <host> that is the target of the <XML-RPC document>.
 
 Description:
 Use the <revXMLRPC_GetHost> <function> to retrieve the host that is the
-target of the execution of an <XML-RPC request>.
+target of the execution of an <XML-RPC> request.
 
 If the <revXMLRPC_GetHost> <function> encounters an error, it
 <return|returns> an error message starting with "xmlrpcerr" ..
 
->*Important:*  The <revXMLRPC_GetHost> <function> is part of the 
-> <XML-RPC library>. To ensure that the <function> works in a 
-> <standalone application>, you must include this 
-> <LiveCode custom library|custom library> when you create your 
-> <standalone application|standalone>. In the Inclusions pane of the 
-> <Standalone Application Settings> window, make sure the "XML-RPC" 
+>*Important:*  The <revXMLRPC_GetHost> <function> is part of the
+> <XML-RPC library>. To ensure that the <function> works in a
+> <standalone application>, you must include this
+> <LiveCode custom library|custom library> when you create your
+> <standalone application|standalone>. In the Inclusions pane of the
+> <Standalone Application Settings> window, make sure the "XML-RPC"
```

Comment:
I note that only a minority of the XML-RPC library syntax dictionary entries have this "Important" paragraph.  However, I'm not sure it's ideal to copy and paste it into every dictionary entry related to the XML-RPC library.  Do you think there's any logical place to just put the warning once, but where most people using the library are likely to see it?  Is there an overall summary page for the XML-RPC library that might be a good place to stash it?